### PR TITLE
Algo added

### PR DESCRIPTION
**Pull request by stark2710**
Do approve

### CODE REVIEW

File: BinarySearch.java [hunk 1]
```diff
@@ -0,0 +1,42 @@
...
+            if (arr[m] < x)
+                l = m + 1;
+ 
+            // If x is smaller, ignore right half
+            else
+                r = m - 1;
+        }
+ 
+        // if we reach here, then element was
+        // not present
```
Comment (Prabhash33) [inline BinarySearch.java:24]: add the comment in a single line

**Review by Guru-S-Vargav — COMMENTED** [review]
Copied
//Plagiarism

**Review by Guru-S-Vargav — APPROVED** [review]

File: BinarySearch.java (hunk 1 above)
Comment (stark2710) [reply BinarySearch.java:24]: not needed cc!

Comment (Prabhash33) [reply BinarySearch.java:24]: achaaa!

**Pull request merged**